app/webpacker/packs/application.js

```diff
@@ -99,3 +99,15 @@ if (stickyMenu) {
     mobileStickySubNav.classList.toggle('app-mobile-nav--active');
   }
 }
+
+$('#content_asset_asset_file').on('change', function() {
+  const size = (this.files[0].size / 1024 /1024).toFixed(2);
+  var $submit = $('input[type="submit"]');
+  $submit.prop('disabled', true);
+  if (size > 2) {
```

Comment:
I would have put 2 * 1024 * 1024 here but hey ho, no worries

Reply:
I literally copied and pasted from a blog post